src/biogeophys/TotalWaterAndHeatMod.F90

```diff
@@ -959,6 +1135,8 @@ subroutine AdjustDeltaHeatForDeltaLiq(bounds, delta_liq, &
     ! liquid and ice runoff in CLM, then this routine should be reworked to use the true
     ! heat contents of both liquid and ice runoff.
     !
+    ! ADD HERE NOTES ABOUT LAKES!!! lake water not accounted. because baselines 
+    !
```

Comment:
Yes, I think I figured out what I wanted to add: 

With dynamical lakes, this routine does not correct for the heat carried within the lake water itself. 
When a lake column grows, extra water content is added to the grid cell, but as water content is expressed in kg/m² and lake depth is constant, the size of the lake does not matter. Because of this, after substracting baselines, the change in lake water content will always be zero.  I guess could be solved once the temperature of runoff is tracked, and lake depths can change too. 

Does this makes senses to you? 

I adjusted the description as follows (but feel free to change or update to make it more clear): 
``` suggestion 
    ! With dynamical lakes, the adjusted delta_heat does not account for the added lake 
    ! water content due to growing lakes. This is because lake depth is constant, the 
    ! total lake water content (kg/m^2) does not change. The change in water content of 
    ! the snow and soil in the lake column are accounted for.`
```

Reply:
Thanks! To be honest, my head isn't enough in this to give this a critical read, but I think this seems good.